.github/workflows/codeql.yml

```
@@ -1,98 +1,130 @@
-        build-mode: ${{ matrix.build-mode }}
-        # If you wish to specify custom queries, you can do so here or in a config file.
-        # By default, queries listed here will override any specified in a config file.
-        # Prefix the list here with "+" to use these queries and those in the config file.
+      # Initialize CodeQL
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v3
+        with:
+          languages: ${{ matrix.language }}
+          build-mode: ${{ matrix.build-mode }}
+          queries:
+            - uses: security-extended
```

actionlint (1.7.7)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)